### PR TITLE
Alerting: Prometheus-compatible Alertmanager timings editor

### LINKED ISSUE #63964: Alerting: adding too large a number in notification policy intervals not managed properly
**What happened**:
If I add a number higher than 60 (seconds) in any of the intervals - group wait, group interval, repeat interval, the counter resets (probably to default - as doing this for all three disabled "override general timings"), as you can see in the screenshot below:
group_interval

**What you expected to happen**:
That the input be parsed/managed somehow. Either explicitly error out or round it up/down or allow it as such.

**Environment**:
- Grafana version: 9.4.1
- OS Grafana is installed on: Kubernetes (Ubuntu image)
- User OS & Browser: Mac OS Ventura 13.2

### PR DESCRIPTION
**Pull request by konrad147**
**What is this feature?**
This PR introduces 100% Prometheus-compatible Alertmanager timing inputs
The previous one didn't allow to configure duration containing multiple time units

Additionally, the duration inputs contain inline documentation explaining the Prometheus duration format

**Why do we need this feature?**
To support all available values for Alertmanager timings

Fixes #63964 

**DEMO**
image

### CODE REVIEW

File: public/app/features/alerting/unified/components/notification-policies/EditNotificationPolicyForm.tsx [hunk 1]
```diff
@@ -50,14 +50,13 @@ export const AmRoutesExpandedForm: FC<AmRoutesExpandedFormProps> = ({ actionButt
...
+  const emptyMatcher = [{ name: '', operator: MatcherOperator.equal, value: '' }];
 
   const receiversWithOnCallOnTop = receivers.sort(onCallFirst);
 
   const formAmRoute = amRouteToFormAmRoute(route);
-
-  const emptyMatcher = [{ name: '', operator: MatcherOperator.equal, value: '' }];
-
-  const defaultValues: FormAmRoute = {
+  // Omit to ignore circular reference error
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/components/notification-policies/EditNotificationPolicyForm.tsx:58]: 💡

**Review by soniaAguilarPeiron — APPROVED** [review]
LGTM!🎉

**Review by gillesdemey — APPROVED** [review]
LGTM, and I love the cleanups here too – much easier to reason about some of this code now 🚀

File: public/app/features/alerting/unified/components/notification-policies/EditDefaultPolicyForm.tsx [hunk 2]
```diff
@@ -106,112 +105,50 @@ export const AmRootRouteForm = ({
...
+              <Field
+                label="Group wait"
+                description="The waiting time until the initial notification is sent for a new group created by an incoming alert. Default 30 seconds."
+                invalid={!!errors.groupWaitValue}
+                error={errors.groupWaitValue?.message}
+                data-testid="am-group-wait"
+              >
+                <PromDurationInput
+                  {...register('groupWaitValue', { validate: promDurationValidator })}
+                  placeholder="30s"
```
[the remark this answers is not in the document]
Comment (konrad147) [reply public/app/features/alerting/unified/components/notification-policies/EditDefaultPolicyForm.tsx:118]: Sure! I wasn't aware of them

File: public/app/features/alerting/unified/components/notification-policies/timingOptions.ts
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/notification-policies/timingOptions.ts:1]: 👏

**Pull request merged**